app/client/src/sagas/ActionExecution/PluginActionSaga/onPageLoadSaga.ts

```
@@ -0,0 +1,397 @@
+      appId: currentApp.id,
+      onPageLoad: true,
+      appName: currentApp.name,
+      environmentId: currentEnvDetails.id,
+      environmentName: currentEnvDetails.name,
+      isExampleApp: currentApp.appIsExample,
+      pluginName: plugin?.name,
+      datasourceId: datasourceId,
+      isMock: !!datasource?.isMock,
+      actionId: pageAction?.id,
+      inputParams: 0,
+      source: !!actionExecutionContext
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)